graphistry/pygraphistry.py

```diff
@@ -347,7 +523,10 @@ def register(
         token_refresh_ms: int = 10 * 60 * 1000,
         store_token_creds_in_memory: Optional[bool] = None,
         client_protocol_hostname: Optional[str] = None,
-        org_name: Optional[str] = None
+        org_name: Optional[str] = None,
+        idp_name: Optional[str] = None,
```

Comment:
when trying to specify `org_name=a, personal_key_id=b, personal_key_secret=c`, it always appears in the personal org vs org a (server version 2.39.30)

I would expect an exception during `register()` call if personal key does not match up to org, vs current silent failure behavior of being allowed to progress and sending data to an incorrect account

Maybe the issue is the server version?

Reply:
https://github.com/graphistry/graphistry/issues/1631